src/nodes.coffee

```diff
@@ -225,6 +225,7 @@ exports.Base = class Base
 # `if`, `switch`, or `try`, and so on...
 exports.Block = class Block extends Base
   constructor: (nodes) ->
+    super()
```

Comment:
ES2015 derived constructors need explicit `super` calls in order to access `this`.